src/main/java/ng/abdlquadri/eventbus/handlers/AsyncResult.java

```diff
@@ -0,0 +1,30 @@
+package ng.abdlquadri.eventbus.handlers;

```

Comment:
Thanks for making the update. However I think we should remove this class from this PR. And add this in a separate PR because I think it will require more thought.